### ISSUE TITLE
PostgreSQL error

### ISSUE THREAD

**Issue by ltx-dev** [netguru-training/wroclaw-ror-workshops-2015-december#1]
I've been wondering if i'm the only one having the following issue:

createuser -s workshops command gives:

psql: could not connect to server: No such file or directory
 Is the server running locally and accepting
 connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?

I've been trying to fix it for past couple of hours with no luck, both on Ubuntu 15.10 and cloud9 environment. None of the stackoverflow solutions seem to work.

Right now it's preventing me from even tackling the project :(

I will be reporting solution immediately if i manage to fix it.

**Comment by mateuszgorniak** [comment]
Hi! :-)
It looks like your problem is caused by database.
On the beginning please verify two things:
- your database is working
- postgres is using proper port

If it's still not working please give me information about ways you tried to fix the problem.

**Comment by pieniek** [comment]
If it's your first time using PostgreSQL did you set password to the user 'postgres' and initialize the database using 'initdb' command and started the postgresql service? 
I did just that and it works. I'm on Arch though so the process may be different for you.

**Comment by ltx-dev** [comment]
Hey a little update. You guys got me seemingly on the right track.

@mateuszgorniak it seems that postgres is complaining about the 5423 port. Solution to that is to change it in the postgres.conf file. However i only happen to have postgres.conf.sample in which i changed the line to allow all ports. Sadly it didn't help, i'm sure i saw postgres.conf in one of the directories i checked but i can't find it anymore.

@Pieniasty Yes it is my first time ( uh huh :)) however at this point i'm pretty sure i screwed something up.
- I started the service (service postgresql)
- su - postgres (Here it asks me for password which i haven't setup)

In other words, it seems to be running but i have no way of accessing it or manipulating config files.

If you guys have any ideas please be sure to let me know, i won't give up.
